src/lib.rs

```diff
@@ -187,8 +187,9 @@ mod zerocopy {
 /// `FromZeroes` for that type:
 /// - If the type is a struct:
 ///   - All of its fields must implement `FromZeroes`
-/// - If the type is an enum, it must follow the same rules as [`FromBytes`]
-///   enums. this will likely change soon.
+/// - If the type is an enum, all of its variants must be fieldless and it must
+///   have a variant with a discriminant of `0` (if the first variant does not
+///   have an explicit discriminant, its discriminant is set to 0).
```

Comment:
How would you feel about removing this parenthetical? IMO, in this position, it feels stylistically like it's describing a requirement of the trait rather than describing a detail of the language. It's also likely the case that a given programmer either already knows about discriminant values and how to set them (and thus the parenthetical isn't telling them something they don't know) or they don't know about how to set discriminant values (in which case they won't know what "explicit discriminant" means). Maybe we could reword this like so?

```rust
/// - If the type is an enum, all of its variants must be fieldless, and it must
///   have a variant with a discriminant of `0` (see [the reference] for
///   a description of how discriminant values are chosen)
...
/// [the reference]: https://doc.rust-lang.org/reference/items/enumerations.html#custom-discriminant-values-for-fieldless-enumerations
```

Also, can you remove the trailing period to be consistent with the other bullet points?

Reply:
Changed wording and removed trailing period